compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEPackToDma.cpp

```diff
@@ -321,6 +321,38 @@ void AMDAIEPackToDmaPass::runOnOperation() {
   MLIRContext *context = &getContext();
   IRRewriter rewriter(context);
 
+  // Convert all linalg.copy to iree_linalg_ext.pack ops.
+  //
+  // This is simple, the iree_linalg_ext.pack is just an identity pack
+  // operation.
+  //
+  // By doing this, we can bootstrap the work done for lowering the pack op to
+  // dmas: easier than writing a new pass to convert linalg.copy to dmas
+  // directly.
+  auto walkCopyToPackResult =
+      getOperation()->walk([&rewriter](linalg::CopyOp copyOp) -> WalkResult {
```

Comment:
This pass happens after bufferization, so it has to be a memref::CopyOp?

Reply:
Also please add some tests.

Reply:
Before comprehensive bufferization:

```
 %16 = linalg.copy ins(%15 : tensor<1x1x4x4xi32>) outs(%extracted_slice_1 : tensor<1x1x4x4xi32>) -> tensor<1x1x4x4xi32>
```

After comprehensive bufferization:

```
linalg.copy ins(%alloc_3 : memref<1x1x4x4xi32, 1 : i32>) outs(%subview_1 : memref<1x1x4x4xi32, strided<[9216, 768, 64, 1], offset: ?>, #hal.descriptor_type<storage_buffer>>)
```

So it's not a memref.copy.